docs/jailer.md

```diff
@@ -168,7 +172,10 @@ Then, redirect standard file descriptors to `/dev/null` by calling
 STDERR)`. Close `dev_null_fd`, because it is no longer necessary.
 
 Finally, the jailer switches the `uid` to `123`, and `gid` to `100`, and execs
-`./firecracker --jailed`. We can now use the socket at
+```
```

Comment:
nit: Code blocks should be separated by empty lines (empty line before and after the code block).

Reply:
Done :)